src/v/debug_bundle/debug_bundle_service.cc

```diff
@@ -12,18 +12,114 @@
 #include "debug_bundle_service.h"
 
 #include "config/configuration.h"
+#include "container/fragmented_vector.h"
 #include "debug_bundle/error.h"
 #include "debug_bundle/types.h"
+#include "security/acl.h"
+#include "ssx/future-util.h"
+#include "utils/external_process.h"
 
 #include <seastar/core/seastar.hh>
 #include <seastar/core/shard_id.hh>
+#include <seastar/core/sstring.hh>
+#include <seastar/util/process.hh>
+#include <seastar/util/variant_utils.hh>
+
+#include <boost/algorithm/string/join.hpp>
+#include <fmt/format.h>
+
+#include <optional>
+
+using namespace std::chrono_literals;
 
 namespace debug_bundle {
 static ss::logger lg{"debug-bundle-service"};
 
+namespace {
+inline bool contains_sensitive_info(const ss::sstring& arg) {
+    if (arg.find("-Xpass") != ss::sstring::npos) {
+        return true;
+    }
+    return false;
+}
+inline void print_arguments(const std::vector<ss::sstring>& args) {
```

Comment:
i think you can drop these `inline` keywords. any reason for them?

Reply:
probably a bad habit of adding them?